Makefile

```diff
@@ -224,7 +226,7 @@ comp:
 	mill -i xiangshan.compile
 	mill -i xiangshan.test.compile
 
-$(call docker-deps,$(TOP_V)): $(SCALA_FILE)
+$(DOCKER_TARGET)$(TOP_V): $(SCALA_FILE)
```

Comment:
unfortunately, there's still no way to completely keep the original makefile un-modified.
we have to choose a more “comfortable” way:
1. add single $(call docker-dep, xxx) to all `targets` needs run in docker
2. add `docker target` to DOCKER_TARGETS variable
    - BUT, still need a $(DOCKER_TARGET) prefix to all `docker target`. otherwise, we will get following makefile warning
      ![image](https://github.com/user-attachments/assets/d65ae228-92ef-40c9-b347-b99b46fb4a3d)

Reply:
@poemonsense @cebarobot this should be the first decision: which way is prefered for modification of original makefile targets

Reply:
I prefer the `call docker-deps` way

Reply:
changed to `$(call docker-deps, xxx)` style
```
-verilog: $(TOP_V)
+verilog: $(call docker-deps,$(TOP_V))
```